Mage.Tests/src/test/java/org/mage/test/cards/continuous/LandTypeChangingEffectsTest.java

```diff
@@ -138,6 +138,7 @@ public void testLandDoesNotLooseOtherAbilities() {
     /*
     NOTE: this test is currently failing due to bug in code. See issue #3072
     */
+    /*
```

Comment:
Why are these tests being removed? They fail currently due to bugs in code and should be preserved until we resolve the issues. If you believe the tests are inaccurate, i.e. an issue with the testing framework itself, then @Ignore annotation is more appropriate than commenting out tests (otherwise we cannot find them again)